packages/addresses/src/__tests__/base58-test.ts

```diff
@@ -132,4 +133,16 @@ describe('base58', () => {
             ]);
         });
     });
+    describe('createBase58AddressWithSeed', () => {
+        it('works', async () => {
```

Comment:
For the benefit of the next reader who doesn't know what this is supposed to do in the first place.

```suggestion
        it('returns an address that is the SHA-256 hash of the concatenated base address, seed, and program address', async () => {
```